docs/proof-of-assets.html

```diff
@@ -28,18 +28,22 @@ <h1 class="large mb64 mb-xs-24">Proof of Assets</h1>
                                     <table class="col-sm-12">
                                         <thead>
                                             <tr>
-                                                <th>Adress</th>
+                                                <th>Type</th>
+                                                <th>Address</th>
                                                 <th>Balance</th>
                                             </tr>
                                         </thead>
                                         <tbody>
-                                            <tr>
-                                                <td><a href="https://blockstream.info/address/1Ni2mG3j5zMyYrJ6gD7XjTgYC4KEeTeniX">1Ni2mG3j5zMyYrJ6gD7XjTgYC4KEeTeniX</a> (Site under construction, not actual assets)</td>
-                                                <td>1234.56</td>
+                                            <tr style="padding-bottom:10px">
+                                                <td style="padding-right:10px">BTC  <br/> (custody)</td>
+                                                <td style="padding-right:10px"><a href="https://blockstream.info/address/1Ni2mG3j5zMyYrJ6gD7XjTgYC4KEeTeniX">{{btcAddress}}</a></td>
```

Comment:
Can we put `{{btcAddress}}` here so the address shown is consistent with the one we link to?